### PR TITLE
Create 'access GUI', 'network/DHCP config' and 'nodes/boot images' docs from installer.

### PR DESCRIPTION
**Pull request by degville**
The content of these files was pulled from the installation document to better fit our new TOC.

### CODE REVIEW

File: src/en/installconfig-gui.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
+Title: Access the GUI
+TODO: Update images
+	
+# Access the GUI
+
+Once MAAS is installed, you'll need to create an administrator account:
+
+```bash
+sudo maas createadmin --username=root --email=MYEMAIL@EXAMPLE.COM
```
Comment (pmatulis) [inline src/en/installconfig-gui.md]: So I believe we should be pushing the GUI by default. All CLI alternatives should be given later. Example:

"To create the superuser vi the CLI see 'Using the CLI#create-superuser' ".

The same goes for all other CLI mentions on this page. We can let it go for now if you want and leave a TODO.

Ditto for page installconfig-images.md

File: src/en/installconfig-network.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
+Title: Network Configuration
```
Comment (pmatulis) [inline src/en/installconfig-network.md]: All this stuff is also in installconfig-gui.md

Comment (degville) [inline src/en/installconfig-network.md]: I think perhaps the best thing to do is for me to raise a couple of issues for these rewrites (to push the GUI) so we can get the new repository populated with the old docs as quickly as possible, before then going back and tackling the issues themselves.

**Comment by degville** [comment]
Merged after opening #21 to deal with the changes we need to make to reference the GUI.

**Pull request merged**